ci/docker/ubuntu.dockerfile

```diff
@@ -20,7 +20,7 @@ ARG NANOARROW_ARCH
 FROM --platform=linux/${NANOARROW_ARCH} ubuntu:latest
 
 RUN apt-get update && DEBIAN_FRONTEND=noninteractive apt-get install -y \
-    locales git cmake r-base gnupg curl valgrind gfortran python3-venv python3-dev doxygen pandoc lcov \
+    locales git cmake r-base gnupg curl valgrind gfortran python3-venv python3-dev doxygen pandoc gcovr \
```

Comment:
I haven't been able to get the docker compose run to work locally - getting an error 

```sh
/coverage.sh: line 88: gcovr: command not found
```

Although I am installing here. Is this not the right place?

Reply:
Ah OK  I see - this registry this pulls from is in the .env file

Reply:
Ah OK cool - just pushed it up using the local image for now. You might want to take a look at the CI output if that's been of value to you in the past. The gcovr output is slightly different than what lcov provides